Sources/PreviewGallery/PreviewData.swift

```diff
@@ -8,22 +8,45 @@
 import Foundation
 import SnapshotPreviewsCore
 
+
+/// A structure that manages a collection of preview types.
+///
+/// `PreviewData` is used to store and manipulate a set of `PreviewType` instances,
+/// providing functionality to filter and sort previews by module.
 public struct PreviewData {
-  let previews: [PreviewType]
+    /// The collection of preview types managed by this instance.
+    let previews: [PreviewType]
 
-  public init(previews: [PreviewType]) {
-    self.previews = previews
-  }
+    /// Initializes a new `PreviewData` instance with the given previews.
+    ///
+    /// - Parameter previews: An array of `PreviewType` instances to be managed by this `PreviewData`.
+    public init(previews: [PreviewType]) {
+        self.previews = previews
+    }
 
-  func previews(in module: String) -> [PreviewType] {
-    previews.filter { $0.module == module }.sorted { $0.typeName < $1.typeName }
-  }
+    /// Retrieves and sorts the previews for a specific module.
+    ///
+    /// This method filters the previews to include only those from the specified module,
+    /// then sorts them alphabetically by their type names.
+    ///
+    /// - Parameter module: The name of the module to filter previews for.
+    /// - Returns: An array of `PreviewType` instances belonging to the specified module, sorted by type name.
+    func previews(in module: String) -> [PreviewType] {
+        previews.filter { $0.module == module }.sorted { $0.typeName < $1.typeName }
+    }
 
-  var modules: Set<String> {
-    Set(previews.map { $0.module })
-  }
+    /// A set of all unique module names represented in the previews.
+    var modules: Set<String> {
+        Set(previews.map { $0.module })
+    }
 
-  @MainActor public static var `default`: PreviewData {
-    self.init(previews: FindPreviews.findPreviews())
-  }
+    /// A default instance of `PreviewData` that automatically finds all available previews.
+    ///
+    /// This property uses `FindPreviews.findPreviews()` to populate the previews.
+    /// It must be accessed from the main actor due to its use of `@MainActor`.
+    ///
+    /// - Note: This property is marked with `@MainActor`, ensuring it's only accessed from the main thread.
```

Comment:
This feels redundant with the line above, and the line above already feels kind of redundant. I'd suggest removing the MainActor stuff from the docs, it's already in the declaration so it's clear